mixins/localize-dynamic-mixin.js

```diff
@@ -0,0 +1,43 @@
+import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
+import { LocalizeMixin } from './localize-mixin.js';
+
+/*** WARNING! ¡ATENCIÓN! ACHTUNG! Adding to defaultLangs MUST be a BREAKING change with a MAJOR version bump! ***/
+const defaultLangs = ['ar', 'cy-gb', 'cy', 'da', 'de', 'en', 'es-es', 'es', 'fr-ca', 'fr', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-tw', 'zh'];
+/****************************************************************************************************************/
+const fallbackLang = 'en';
+const defaultExportName = 'default';
+
+export const LocalizeDynamicMixin = superclass => class extends LocalizeMixin(superclass) {
+
+	static get config() {
+		return {};
+	}
+
+	static async getLocalizeResources(langs, config) {
+		const { importFunc, osloCollection, supportedLangs = defaultLangs, exportName = defaultExportName } = config;
+
+		for (const lang of [...langs, fallbackLang]) {
+
+			if (supportedLangs.includes(lang)) {
+				const mod = await importFunc(lang).catch(() => {});
```

Comment:
Some people find the `await/catch()` mixture objectionable, but it's cleaner than a `try/catch`. This will only supress the javascript errors, not the initial network error.

Reply:
At least this will suppress the JS error like you said, and it'll allow the loop through the supported languages to continue... which likely just means it'll fall back to English. But we also added `es-es` recently so it would have nicely fallen back to `es`.

Would a `console.warn` inside the `catch` bit make sense? Not sure if we can get the message to be meaningful enough for devs to actually be able to debug it though... like it would need to have the element name I think and I'm not sure we can get that.

Reply:
Since the actual error occurs on the arrow function that the components write, it should be in the stack.